google/cloud/spanner_v1/_opentelemetry_tracing.py

```diff
@@ -72,6 +79,15 @@ def trace_call(name, session, extra_attributes=None):
     if extra_attributes:
         attributes.update(extra_attributes)
 
+    # TODO(@odeke-em) enable after discussion with team and agreement
+    # over extended tracing changes as the legacy default is always to
+    # record SQL statements on spans, because the prior behavior was
+    # to always record the SQL statement and changing it is 's considered
+    # a breaking change by the Python-Spanner team.
+    if False and not enable_extended_tracing:
+        attributes.pop("db.statement", False)
+        attributes.pop("sql", False)
```

Comment:
what is this attribute? How does it help?

Reply:
It was the legacy variable, but I've removed it.